mne/viz/evoked.py

```diff
@@ -2304,29 +2304,34 @@ def click_func(
             evokeds, conditions, scaling, [picks_], ci_fun, gfp)
         # we now have dicts for data ('evokeds' - grand averaged Evoked's)
         # and the CI ('ci_array') with cond name labels
-#        return data_dict, ci_dict
         all_data.append(data_dict)
         all_cis.append(ci_dict)
     del evokeds
 
     if do_topo:
+        # ax_anchors are left, bottom coords of axis (in figure coords)
         ax_anchors = np.array([np.array(ax_.get_position())[0]
                                for ax_, _ in axes])
         left, bottom = ax_anchors[:, 0].max(), ax_anchors[:, 1].min()
-        (x0, y0), (x1, y1) = np.array(ax.get_position())
-        ax = plt.axes([left, bottom + .05, x1 - x0, y1 - y0])
-        axes.append((ax, -1))
-        picks.append(0)
+        # add an axis for the legend
+        (lef, bot), (wid, hei) = np.array(ax.get_position())
+        legend_ax = plt.axes([left, bottom + .05, wid - lef, hei - bot])
+        axes.append((legend_ax, -1))
+        picks.append(-1)
 
+    # add empty data (all zeros) for the legend axis
     all_data.append({cond: np.zeros(d.shape) for cond, d in data_dict.items()})
     all_cis.append({cond: np.zeros(np.array(d).shape)
                     for cond, d in ci_dict.items()})
 
     for picks_, (ax, idx), data, cis in zip(picks, axes, all_data, all_cis):
         if do_topo:
             title = all_ch_names[idx]
+        if idx == -1:
+            # do something here to suppress drawing of fake data on the legend
+            pass
```

Comment:
TODO

Reply:
Then you can delete the part later where the lines are manually deleted right?

`ax.lines.clear()` in 2372

Reply:
hmm...  at least in my local testing `ax.lines.clear()` wasn't actually clearing the lines (I didn't even notice it).

Reply:
Then it can be doubly deleted :D

Reply:
Eh I think it did delete the lines cause the unit test relies on it. But the unit test should still pass with this instead.